### DISCUSSION TITLE
Compile Error With Latest Merge

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#1140] [General Discussion]
@Jyers Using 3X3 Manual Mesh template. Error when compiled.

![Error](https://user-images.githubusercontent.com/76499230/125217776-7c631100-e28f-11eb-95a6-e8f1f3f9bcdd.jpg)

**Comment by Jyers** [comment]
The one template I didn't check...

I'll fix it

**Reply by Jyers** [reply]
Done.

**Comment by adam3654** [comment]
This is new, I've never had a choice of environments before 
![image](https://user-images.githubusercontent.com/16920869/125225357-2ac18300-e29d-11eb-9bdf-154fa4c481d9.png)

**Reply by Jyers** [reply]
Yeah they just added the maple environment. Not sure what hardware it's for though.

**Reply by adam3654** [reply]
I can't seem to get it to compile now, I am working off of your latest commit, here is the output:
![image](https://user-images.githubusercontent.com/16920869/125228526-e507b900-e2a2-11eb-9373-5c8b4be621e0.png)

**Reply by Jyers** [reply]
Perhaps try a fresh clone, looks like you might have some old files there.

**Reply by adam3654** [reply]
I got it to work with a fresh clone, then manually working through the config files to adjust values (thanx Notepad++). 
Something I noticed though... The file size is 186KB, vs 210-ish pre merge. Is that right or did I mess up the config file. I'm kinda scared to upload it now, LOL.

**Reply by Cal-LF** [reply]
@adam3654 My file size is smaller also. 184k now 159k. Firmware installed and functions all working.

**Reply by adam3654** [reply]
Thanks I was able to install it late last night, everything seems to be working other than manual leveling.

**Reply by Cal-LF** [reply]
@adam3654 What is happening with the manual leveling? Do you mean the 4 probe points are not working? Manual level or probe assisted tramming?

**Reply by adam3654** [reply]
I'm getting the same results as described here:

https://github.com/Jyers/Marlin/issues/1125#issuecomment-878499891

**Reply by Cal-LF** [reply]
Oh, so the probe assist tramming is not working.

**Reply by adam3654** [reply]
> Oh, so the probe assist tramming is not working.

Probe assist tramming would just be the "Use Probe" option in Manual Leveling, right? If so, then yes. I've never been able to get it working properly. I just paper level it and run a 15x15 mesh after I mess with the setup in a way that could knock the plate out of level. 

I really do want to get the assisted tramming working though, it looks like I could really dial in the plate with some hard numbers to work with.

**Reply by AshenColors** [reply]
> I can't seem to get it to compile now, I am working off of your latest commit, here is the output:
> ![image](https://user-images.githubusercontent.com/16920869/125228526-e507b900-e2a2-11eb-9373-5c8b4be621e0.png)

If anyone else runs into this, it's referenced upstream here. The takeaway is that some stuff gets renamed and PIO gets confused, so deleting the .pio folder is sufficient instead of making an entirely new clone, or at least that was the case for me.
https://github.com/MarlinFirmware/Marlin/issues/22160#issuecomment-863509956

**Reply by Cal-LF** [reply]
@AshenColors Make sure you are using JyresUI, the default commit at the top of the branch list. Use one of the templates in the E3V2 config files.
